frontend/apps/web/app/(mgmt)/[account]/connections/[id]/components/AwsS3Form.tsx

```diff
@@ -80,223 +80,227 @@ export default function AwsS3Form(props: Props) {
     }
   }
   return (
```

Comment:
hide whitespace in the PR to get a better sense of what changed here.
I changed some nesting and reduced the spacing that was added as a part of the form error change to squish the whitespace a bit more.